Documentation/network/servicemesh/grpc.rst

```diff
@@ -35,6 +35,33 @@ Deploy GRPC Ingress
 
 You'll find the example Ingress definition in ``example/kubernetes/servicemesh/grpc-ingress.yaml``.
 
+.. code-block:: yaml
```

Comment:
Maybe we can use `.. include::`  to directly include the examples from the `examples/` directory instead, so that the docs are always in sync with the example YAMLs?

Reply:
Good point, I didn't know about literalinclude directive before :100: